testdata/generated/package/input_groups/0.0.1/index.json

```diff
@@ -0,0 +1,293 @@
+{
+  "name": "aws",
+  "title": "AWS",
+  "version": "0.0.1",
+  "release": "beta",
+  "description": "AWS Integration",
+  "type": "integration",
+  "download": "/epr/aws/aws-0.0.1.zip",
+  "path": "/package/aws/0.0.1",
+  "icons": [
+    {
+      "src": "/img/logo_aws.svg",
+      "path": "/package/aws/0.0.1/img/logo_aws.svg",
+      "title": "logo aws",
+      "size": "32x32",
+      "type": "image/svg+xml"
+    },
+    {
+      "src": "/img/logo_ec2.svg",
```

Comment:
Why does the ec2 logo show up in 2 places?

Reply:
Fixed now 👍